test/extended/authorization/rolebinding_restrictions.go

```diff
@@ -0,0 +1,233 @@
+package authorization
+
+import (
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kapi "k8s.io/kubernetes/pkg/apis/core"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/apis/authorization"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Feature: RoleBinding Restrictions][Serial] RoleBindingRestrictions should be functional", func() {
```

Comment:
Added [Serial] bc when run parallel random test cases fail (not same ones every time).  I need to investigate this more.

Reply:
With my local cluster of the day, removed 'Serial' and tests are passing.  I'm seeing `err:"net/http: request canceled (Client.Timeout exceeded while reading body)"` however, so maybe w/ parallel tests will be flaky? running through CI to investigate.

Reply:
passed thru CI as parallel,once this merges i'll monitor for flakes